pennylane/utils.py

```diff
@@ -353,3 +356,30 @@ def __str__(self):
                 output += "{}({}(wires={}))\n".format(return_map[op.return_type], op.name, op.wires)
 
         return output
+
+
+def inv(operation_list):
+    """Invert a list of operations or a template.
+
+    If the inversion happens inside a QNode, the operations are removed and requeued
+    in the reversed order for proper inversion.
+
+    Args:
+        operation_list (Sequence[~.Operation]): A sequence of operations
+
+    Returns:
+        Sequence[~.Operation]: The inverted sequence of operations
+    """
```

Comment:
It might be worth including a check for ``operation_list`` being a list (else throwing an error).

Reply:
I included checks on the types!